Samples/Desktop/D3D12Raytracing/src/D3D12RaytracingProceduralGeometry/Raytracing.hlsl

```diff
@@ -90,6 +90,39 @@ float4 CalculatePhongLighting(in float4 albedo, in float3 normal, in bool isInSh
     return ambientColor + diffuseColor + specularColor;
 }
 
+// Load three 16 bit indices.
```

Comment:
Does the sample fail to compile with the helper function being the helper? If not, this is not desired. This was put to helper as the samples move code introduced in previous samples and code that is not of the primary goal of the sample to helpers to make the new sample focused on what it's trying to show and keep the code footprint small(er).